openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -597,6 +618,312 @@ Location: https://client.example.net/cb?
+The response to an Interactive Authorization Request is an HTTP message with the content type `application/json` and a JSON document in the body that indicates either
+
+ 1. that user interaction is required, either a Presentation or a custom interaction, as defined in (#iar-interaction-required-response), or
+ 2. a successful completion of the authorization, as defined in (#iar-authorization-code-response), or
+ 3. an error as defined in Section 2.3 of [@!RFC9126].
+
+Except in error cases, the following key is required in the JSON document of the response:
+
+* `status`: REQUIRED. String indicating whether an additional interaction is required or the authorization has been completed.
+
+Depending on this assessment, the response from the Interactive Authorization Endpoint can take one of the following forms:
+
+### Interaction Required Response {#iar-interaction-required-response}
+By setting `status` to `require_interaction` in the response, the Authorization Server requests an additional user interaction.
+In this case, the following keys MUST be present in the response as well:
+
+* `type`: REQUIRED. String indicating which type of interaction is required, as defined below. The Authorization Server MUST NOT set this to a value that was not included in the `interaction_types_supported` parameter sent by the Wallet.
+* `auth_session`: REQUIRED. String containing a value that allows the Authorization Server to associate subsequent requests by this Wallet with the ongoing authorization request sequence. Wallets SHOULD treat this value as an opaque value.
+
+The Wallet MUST include the `auth_session` in all follow-up requests to the Interactive Authorization Endpoint.
+If, as a response to such a follow-up request, the Wallet receives an `auth_session` value that differs from the one sent in the request, it MUST abort the issuance process.
+
+If a wallet receives a `type` value that it does not recognize, it MUST abort the issuance process.
+
+Additional keys are defined based on the type of interaction, as shown next.
+
+#### Require Presentation {#iar-require-presentation}
+
+If `type` is set to `openid4vp_presentation`, as shown in the following example, the response MUST further include an `openid4vp_presentation` parameter containing an OpenID4VP Authorization Request, either in unsigned form as defined in Appendix A.3.1 of [@!OpenID4VP] or signed as defined in Appendix A.3.2 of [@!OpenID4VP].
+
+The following is a non-normative example of an unsigned Authorization Request:
+
+```
+HTTP/1.1 200 OK
+Content-Type: application/json
+Cache-Control: no-store
+
+{
+  "status": "require_interaction",
+  "type": "openid4vp_presentation",
+  "auth_session": "wxroVrBY2MCq4dDNGXACS",
+  "openid4vp_presentation": {
+    "client_id": "x509_san_dns:rp.example.com",
```

Comment:
I think (as this is an unsigned request) `client_id` should actually be omitted.

```suggestion
```